rules/PhpSpecToPHPUnit/PHPUnitTypeDeclarationDecorator.php

```diff
@@ -43,7 +43,7 @@ public function decorate(ClassMethod $classMethod): void
         $returnType = $reflectionMethod->getReturnType();
         $returnTypeName = $returnType instanceof ReflectionNamedType
             ? $returnType->getName()
-            : (string) $returnType;
+            : $returnType->getName();
```

Comment:
this seems changed by `ChangeReflectionTypeToStringToGetNameRector` which should not as the `getName()` only in `ReflectionNamedType`, this possibly related to the error on php 7.0 scoped.

Reply:
Temporary skip `ChangeReflectionTypeToStringToGetNameRector` on rector.php https://github.com/rectorphp/rector-src/pull/199/commits/ced3b8e8b440dc612bb8eea1d59912aa05f5c158